src/controllers/departmentDetails.controller.js

```diff
@@ -0,0 +1,29 @@
+const { departmentService, departmentDetailsService } = require('../services');
+const { error, success } = require('@yapsody/lib-handlers');
+
+const { departmentValidation,getId,getListValidation,updateDepartmentValidation,recoveryParamsValidation, departmentDetailsValidation} = require('../validations');
+const { version } = require('chai');
+
+
+//create Department
+  const addDetails = async (req ,res ,next) => {
```

Comment:
adddetails is not appropiate name 
if your function is for adding department detail, you can name it to 
addDepartmentdetails